recipes/gnutls/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "gnutls" %}
+{% set version = "3.5.15" %}
+{% set sha256 = "046081108b8b1fe455a13a4c5a4eaa0368e185b678f1670fe09a11a2d7ecfad5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: ftp://ftp.gnutls.org/gcrypt/{{ name }}/v3.5/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    - pkgconfig.patch
+
+
+build:
+  number: 0
+  skip: True  # [win]  
+
+requirements:
+  build:
+    - gmp 6.1.*
+    - nettle
```

Comment:
`nettle` needs to be pinned

Reply:
Ok. I am not sure I understand the principel of pinning though. Should nettle be included in this file then ? https://github.com/conda-forge/conda-forge.github.io/blob/fbd5063c2ddac6f81fef061557de27f71f0827c6/scripts/pin_the_slow_way.py#L50

Reply:
Yes, it should be included there. From https://abi-laboratory.pro/tracker/timeline/nettle/ it looks like we need something like, `nettle 3.3|3.3.*`. Can you open a PR at `conda-forge.github.io` as well?

Reply:
Ok I will do it.